content/pt/docs/concepts/cluster-administration/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Cluster Administration"
```

Comment:
Can you translate to Portuguese this title, please?

Reply:
Nice, thanks